axelrod/strategies/axelrod_first.py

```diff
@@ -56,97 +69,202 @@ def strategy(self, opponent: Player) -> Action:
         opponent ever plays D."""
         if len(self.history) < self._rounds_to_cooperate:
             return C
-        if opponent.defections:
+        if opponent.defections > 0:
             return D
         return C
 
+class FirstByDowning(Player):
```

Comment:
Since RevisedDowning is in the second tournament, can we also preserve that implementation and/or compare to the fortran implementation?

Reply:
Sorry I meant to write that in my PR: we need to consider what we do with `RevisedDowning`. As it's in the second tournament is it worth waiting until we translate https://github.com/Axelrod-Python/TourExec/blob/v0.2.0/src/strategies/K59R.f or implement `RevisedDowning` as a modification of the strategy in this PR?

Reply:
The current `RevisedDowning` looks really similar to the Fortran code. If they are basically the same I'm in favor of leaving `RevisedDowning` (eliminating the `revised` bool) as the second tournament implementation. Maybe comparing the fingerprints will tell us if it's essentially correct or not?

Reply:
Yeah good call.

I'm struggling to get `axelrod_fortran` to work on my current machine (I blame an OS update), could you or @meatballs **if you get time** paste fingerprints for "k59r".

Something like:

```
import axelrod as axl
import axelrod_fortran as axlf

downing = axlf.Player("k59r")
ashlock_fp = axl.AshlockFingerprint(strategy=downing)
data = ashlock_fp.fingerprint()  # This will take a little while
p = ashlock_fp.plot()
p.savefig("k59r_ashlock_fingerprint.png")

transitive_fp = axl.TransitiveFingerprint(strategy=downing)
data = transitive_fp.fingerprint() 
p = transitive_fp.plot()
p.savefig("k59r_transitive_fingerprint.png")
```

Here are the equivalent for `RevisedDowning`:

Ashlock:

![downing_ashlock_fingerprint](https://user-images.githubusercontent.com/2131546/70075845-7b6af780-15f5-11ea-87c9-590028a27069.png)



Transitive:

![downing_transitive_fingerprint](https://user-images.githubusercontent.com/2131546/70075863-83c33280-15f5-11ea-858d-48e8e1178240.png)

Reply:
They are really similar but not 100% identical

[transative](https://imgur.com/88mYf5b)

[ashlock](https://imgur.com/nxBTUTq)

Reply:
Gosh they're incredibly similar though. I'm happy and looking at the history of `RevisedDowning` you implemented it from the Fortran code and I suspect you did it right.

Reply:
Let's keep it and then tweak it in a follow-up PR

Reply:
Fine by me. :+1: